ddtrace/contrib/logbook/__init__.py

```diff
@@ -0,0 +1,60 @@
+r"""
+Datadog APM traces can be integrated with the logs produced by ```logbook`` by:
+
+1. Having ``ddtrace`` patch the ``logbook`` module. This will configure a
+patcher which appends trace related values to the log.
+
+2. Ensuring the logger has a format which emits new values from the log record
+
+3. For log correlation between APM and logs, the easiest format is via JSON
+so that no further configuration needs to be done in the Datadog UI assuming
+that the Datadog trace values are at the top level of the JSON
+
+Enabling
+--------
+
+Patch ``logbook``
+~~~~~~~~~~~~~~~~~~~
+
+If using :ref:`ddtrace-run<ddtracerun>` then set the environment variable ``DD_LOGS_INJECTION=true``.
+
+Or use :func:`patch()<ddtrace.patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(logbook=True)
+
+Proper Formatting
+~~~~~~~~~~~~~~~~~
+
+The trace values are patched to every log at the top level of the record. In order to correlate
+logs, it is highly recommended to use JSON logs which can be achieved by using a handler with
+a proper formatting::
+
+    handler = FileHandler(format_string='{{\"message\": "{record.message}", '
+                                        '\"dd.trace_id\": "{record.dd.trace_id}", '
+                                        '\"dd.span_id\": "{record.dd.span_id}", '
+                                        '\"dd.env\": "{record.dd.env}", '
+                                        '\"dd.service\": "{record.dd.service}", '
+                                        '\"dd.version\": "{record.dd.version}"}}')
```

Comment:
We should update this to reflect the new format

Reply:
Done